### PR TITLE
Add support for wemeet

### PR DESCRIPTION
**Pull request by EricLin0509**
Wemeet (aka. Tencent Meeting) is an online meeting application made by Tencent

### CODE REVIEW

File: apparmor.d/profiles-s-z/wemeet [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+profile wemeet @{exec_path} flags=(attach_disconnected) {
+    include <abstractions/base>
+    include <abstractions/nameservice-strict>
+    include <abstractions/common/bwrap>
+    include <abstractions/common/chromium>
+    include <abstractions/X-strict>
+    include <abstractions/mesa>
+    include <abstractions/vulkan-strict>
+    include <abstractions/dri>
+    include <abstractions/desktop>
```
Comment (roddhjav) [inline apparmor.d/profiles-s-z/wemeet:21]: `X-strict` is already included in `desktop`

File: apparmor.d/profiles-s-z/wemeet [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    network inet6 dgram,
+    network inet6 stream,
+
+    @{sh_path} r,
+    @{bin}/basename rix,
+    @{bin}/bwrap rix,
+    @{bin}/id rix,
+    @{bin}/mkdir rix,
+
+    @{exec_path} mr,
```
Comment (roddhjav) [inline apparmor.d/profiles-s-z/wemeet:38]: Move `@{exec_path} mr,` after the network rules.

File: apparmor.d/profiles-s-z/wemeet [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    /usr/share/hwdata/pnp.ids r,
+    /usr/share/icons/{,**} r,
+    /usr/share/mime/{,**} r,
+    /etc/machine-id r,
+    /usr/share/ca-certificates/trust-source/{,**} r, 
+    /etc/ca-certificates/extracted/tls-ca-bundle.pem r,
+    /etc/ca-certificates/trust-source/{,**} r,
```
Comment (roddhjav) [inline apparmor.d/profiles-s-z/wemeet:46]: Use `abstractions/ssl_certs` instead.

File: apparmor.d/profiles-s-z/wemeet [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    @{bin}/basename rix,
+    @{bin}/bwrap rix,
+    @{bin}/id rix,
+    @{bin}/mkdir rix,
+
+    @{exec_path} mr,
+
+    /usr/share/hwdata/pnp.ids r,
+    /usr/share/icons/{,**} r,
+    /usr/share/mime/{,**} r,
```
Comment (roddhjav) [inline apparmor.d/profiles-s-z/wemeet:42]: Remove, they are already included in `desktop`

File: apparmor.d/profiles-s-z/wemeet [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    /etc/machine-id r,
+    /usr/share/ca-certificates/trust-source/{,**} r, 
+    /etc/ca-certificates/extracted/tls-ca-bundle.pem r,
+    /etc/ca-certificates/trust-source/{,**} r,
+    /var/cache/ w,
+
+    # wemeet specifics
+    /opt/wemeet/bin/** rix,
```
Comment (roddhjav) [inline apparmor.d/profiles-s-z/wemeet:50]: Remove this comment and move this rule with the other `rix` rules.

File: apparmor.d/profiles-s-z/wemeet [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    owner @{user_share_dirs}/wemeetapp/{,**} rwk,
```
Comment (roddhjav) [inline apparmor.d/profiles-s-z/wemeet:51]: That should be:
```
  owner @{user_share_dirs}/wemeetapp/ rw,
  owner @{user_share_dirs}/wemeetapp/** rwlk -> @{user_share_dirs}/wemeetapp/**,
```

File: apparmor.d/profiles-s-z/wemeet [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    @{PROC}/@{pid}/stat r,
+    @{PROC}/@{pid}/statm r,
+    @{PROC}/sys/fs/inotify/max_user_watches r,
+    owner @{PROC}/@{pid}/cmdline r,
+
+    /dev/ r,
+    /dev/tty rw,
+    /dev/shm/ r,
+    /dev/pts/[0-9]* rw,
+
```
Comment (roddhjav) [inline apparmor.d/profiles-s-z/wemeet:66]: Add the local include:
```
  include if exists <local/wemeet>
```

File: apparmor.d/profiles-s-z/wemeet [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+    @{PROC}/@{pid}/net/wireless r,
+    @{PROC}/@{pid}/stat r,
+    @{PROC}/@{pid}/statm r,
+    @{PROC}/sys/fs/inotify/max_user_watches r,
+    owner @{PROC}/@{pid}/cmdline r,
+
+    /dev/ r,
+    /dev/tty rw,
+    /dev/shm/ r,
+    /dev/pts/[0-9]* rw,
```
Comment (roddhjav) [inline apparmor.d/profiles-s-z/wemeet:65]:
```suggestion
    /dev/pts/@{int} rw,
```

**Comment by EricLin0509** [comment]
OK, the issue should be fixed now

**Pull request merged**